src/test/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImplTest.java

```diff
@@ -49,10 +52,12 @@
 	public void init() {
 		MockitoAnnotations.initMocks(this);
 		message = TestUtilMessageService.getArtifactResponseMessage();
```

Comment:
Why making 2 calls, one without token and another with token, and storing result in same variable?

Reply:
I will fix that

Reply:
Fix ok :)